elasticsearch/pom.xml

```diff
@@ -54,6 +54,31 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
> But I didn't add anything new to the local settings, I did rebuild ES from sources locally though.

I see. That's a portability issue for the build :(

> How else are you testing with the right ES version? the one compatible with Lucene 5.2 is not released yet..

I strive for not depending on a specific ES version. Atm. I am testing with the latest published final one (1.7.1) but it should also work with ES 2 as long as their REST API is stable. Which Lucene version it internally happens to use should not bother us. At some point we may add test runs for a range of ES versions, but I it's needed for now IMO.